public/data/python.json

```diff
@@ -239,6 +578,131 @@
         ],
         "tags": ["python", "math", "prime", "check"],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Generate Fibonacci Sequence",
+        "description": "Generates a Fibonacci sequence up to a given number of terms.",
+        "code": [
+          "def generate_fibonacci(n):",
+          "    sequence = [0, 1]",
+          "    for _ in range(2, n):",
+          "        sequence.append(sequence[-1] + sequence[-2])",
+          "    return sequence[:n]",
+          "",
+          "# Usage:",
+          "print(generate_fibonacci(10))  # Output: [0, 1, 1, 2, 3, 5, 8, 13, 21, 34]"
+        ],
+        "tags": ["python", "math", "fibonacci", "sequence"],
+        "author": "axorax"
+      },
+      {
+        "title": "Check Perfect Square",
+        "description": "Checks if a number is a perfect square.",
+        "code": [
+          "def is_perfect_square(n):",
+          "    if n < 0:",
+          "        return False",
+          "    root = int(n**0.5)",
+          "    return root * root == n",
+          "",
+          "# Usage:",
+          "print(is_perfect_square(16))  # Output: True",
+          "print(is_perfect_square(20))  # Output: False"
+        ],
+        "tags": ["python", "math", "perfect square", "check"],
+        "author": "axorax"
+      },
+      {
+        "title": "Convert Binary to Decimal",
+        "description": "Converts a binary string to its decimal equivalent.",
+        "code": [
+          "def binary_to_decimal(binary_str):",
+          "    return int(binary_str, 2)",
+          "",
+          "# Usage:",
+          "print(binary_to_decimal('1010'))  # Output: 10",
+          "print(binary_to_decimal('1101'))  # Output: 13"
+        ],
+        "tags": ["python", "math", "binary", "decimal", "conversion"],
+        "author": "axorax"
+      },
+      {
+        "title": "Generate Random Number",
+        "description": "Generates a random number within a given range.",
+        "code": [
+          "import random",
+          "",
+          "def generate_random_number(start, end):",
+          "    return random.randint(start, end)",
```

Comment:
As this just wraps over the function of a lib without adding any new functionnality, I don't think if fits here.